app/models/item.rb

```diff
@@ -38,4 +37,12 @@ def self.items_by_popularity(number, order = 'desc')
     .limit(number)
   end
 
+  def loaded_image?
+    begin
+      open(self.image)
+    rescue
+      self.errors.add(:image)
+      false
+    end
```

Comment:
WHOA. I don't think I've seen begin/rescue before! I need to research those.

Reply:
This is my first time using those!